test/Microsoft.AspNet.Diagnostics.Tests/project.json

```diff
@@ -3,6 +3,7 @@
         "warningsAsErrors": "true"
     },
     "dependencies": {
+        "Microsoft.AspNet.Buffering": "1.0.0",
```

Comment:
If you check this dependency in we'll have to add the dependency in the CI graph.